importer/api/__init__.py

```diff
@@ -36,17 +37,22 @@ def import_course_from_file(filename, user_id):
     except ArchiveException:
         remove(filename)
         raise ValueError("Invalid OLX archive, unable to extract.")
-    dirs = glob(join(tempdir, "*"))
-    if len(dirs) != 1:
-        rmtree(tempdir)
-        remove(filename)
-        raise ValueError("Invalid OLX archive, bad directory structure.")
-    import_course_from_path(dirs[0], user_id)
+    course_imported = False
```

Comment:
@pwilkins: agreed, we should probably be that specific, since that does reflect the real-world.

![code commends](https://pbs.twimg.com/media/B3qIJLFCcAEJLWm.jpg)